sites/platform/src/languages/php/swoole.md

```diff
@@ -11,11 +11,16 @@ Unlike PHP-FPM’s stateless operating, Swoole relies on establishing persistent
 [Swoole](https://github.com/swoole/swoole-src) and [Open Swoole](https://openswoole.com/) are two forked libraries pursuing that goal.
 
 {{< note >}}
-Swoole requires PHP 7.3+.
-
-The Swoole installation script is compatible up to PHP 8.0.
+The `swoole` and `openswoole` extensions are [available by default](/languages/php/extensions.md) on PHP 8.2+ {{% vendor/name %}} containers.
```

Comment:
As of today, it's only PHP 8.2.

Reply:
@thomasdiluccio Facepalm, thank you (and I need to test if the installation script works with PHP 8.3)